solax/inverter.py

```diff
@@ -367,8 +373,8 @@ class X1Mini(InverterPost):
         vol.Required('Information'): vol.Schema(
             vol.All(
                 vol.Length(min=9, max=9)
-                )
```

Comment:
Could you please remove these white-space changes? I think it would just need a few spaces for the diff to be identical